client/src/components/course-view/CoursePartsView.tsx

```diff
@@ -323,6 +331,12 @@ const CoursePartsView = (): JSX.Element => {
       : []),
   ];
 
+  const confirmDiscard = async (): Promise<void> => {
+    if (await AsyncConfirmationModal({confirmNavigate: true})) {
```

Comment:
The title and text for the modal is missing.

Reply:
The text for the modal is missing in many of the discard modals, 
was it intended to have a better default text for the modal than "AsyncConfirmation Modal Title"? Or is it better to add text to every single one?
Otherwise thanks for implementing a better modal system!

Reply:
For confirming navigation the body and title can always be the same. I tried to implement a default but forgot to test it. Should be fixed now.

The defaults only trigger when `confirmNavigate=true`, not sure if we want to for example throw an error if the title or message is missing otherwise.